### DISCUSSION TITLE
[ACTEUR] Founirsseur de Ressource

### DISCUSSION THREAD

**Discussion by WITNicolas** [witsa/synapps#518] [Ideas]
Le fournisseur de ressource global est maintenant en relatif à la création dans les Globales. (evolution de 1.1.2)
L'acteur fournisseur de ressource dans une scene ne devrait-il pas suivre la même logique ? A priori il se cré en mode Automatique

![image](https://user-images.githubusercontent.com/33632664/176924453-655e3eac-9b9d-4345-8591-e23b7ba1a6f4.png)

**Comment by manufitoussiwit** [comment]
Disons que lorsqu'on ajoute un fournisseur dans une scène, il y a de très grandes chances qu'on l'utilise sur place. Ce n'est pas le cas des globaux qui ne sont pas forcément utilisés sur toutes les scènes.

Aussi, un fournisseur en mode relatif ne fera aucune requête tant qu'il n'y a aucun relatif sur lui. Donc, si on veut utiliser sa donnée directement, elle sera absente.

**Reply by WITNicolas** [reply]
Ok reçu. Merci
